package proto_ff;

import "yd_fieldoptions.proto";


message artifactartifactattributeDesc
{
	optional int32 type = 1[(yd_fieldoptions.field_cname) = "ID"];
	optional int32 value = 2[(yd_fieldoptions.field_cname) = "值"];
}

message artifactartifact
{
	optional int64 id = 1[(yd_fieldoptions.field_cname) = "神器ID"];
	optional string resource = 2[(yd_fieldoptions.field_cname) = "资源形象", (yd_fieldoptions.field_bufsize) = 60];
	optional string previewRes = 3[(yd_fieldoptions.field_cname) = "展示资源", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 professionID = 4[(yd_fieldoptions.field_cname) = "职业ID"];
	optional string name = 5[(yd_fieldoptions.field_cname) = "名字", (yd_fieldoptions.field_bufsize) = 60];
	optional string activeDesc = 6[(yd_fieldoptions.field_cname) = "激活描述", (yd_fieldoptions.field_bufsize) = 140];
	optional int32 quality = 7[(yd_fieldoptions.field_cname) = "品质"];
	optional int64 starId = 8[(yd_fieldoptions.field_cname) = "升星道具id"];
	optional string starNum = 9[(yd_fieldoptions.field_cname) = "升星道具数量", (yd_fieldoptions.field_bufsize) = 200];
	optional int32 starUp = 10[(yd_fieldoptions.field_cname) = "升星等级上限"];
	optional int32 starBer = 11[(yd_fieldoptions.field_cname) = "每次升星属性增加的万分比值"];
	optional int64 skill = 12[(yd_fieldoptions.field_cname) = "技能id"];
	optional string skillUi = 13[(yd_fieldoptions.field_cname) = "技能描述", (yd_fieldoptions.field_bufsize) = 140];
	optional int32 artifactType = 14[(yd_fieldoptions.field_cname) = "神器种类"];
	repeated artifactartifactattributeDesc attribute = 15[(yd_fieldoptions.field_cname) = "基础属性", (yd_fieldoptions.field_arysize) = 5];
}


message Sheet_artifactartifact
{
	repeated artifactartifact artifactartifact_List = 1[(yd_fieldoptions.field_arysize)=40];
}
